darts/models/forecasting_model.py

```diff
@@ -49,13 +49,15 @@ def __init__(self):
 
     @abstractmethod
     def fit(self, series: TimeSeries) -> None:
-        """ Trains the model on the provided series
+        """ Trains the model on the provided univariate series.
 
         Parameters
         ----------
         series
             A target time series. The model will be trained to forecast this time series.
         """
+        if not isinstance(self, ExtendedForecastingModel):
```

Comment:
The reason I'm not is because `GlobalForecastingModel` does not call the constructor of its superclass in this implementation. Do you think that's ok?

Reply:
Yes, but perhaps it wouldn't hurt to add `GlobalForecastingModel` to the check here, in case we change it one day and call the init method.